src/TSMapEditor/Models/CliffType.cs

```diff
@@ -108,7 +108,7 @@ public CliffAStarNode(CliffAStarNode parent, CliffConnectionPoint exit, Point2D
         /// Distance to end node
         /// </summary>
         public float HScore => Vector2.Distance(Destination.ToXNAVector(), ExitCoords.ToXNAVector());
-        public float FScore => GScore * 0.7f + HScore;
+        public float FScore => GScore * 0.7f + HScore + Tile?.DistanceModifier ?? 0;
```

Comment:
I think it would be good to clarify the operation order here.

```suggestion
        public float FScore => GScore * 0.7f + HScore + (Tile?.DistanceModifier ?? 0);
```